cmd/sonobuoy/app/run.go

```diff
@@ -88,15 +96,6 @@ func submitSonobuoyRun(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 
-	plugins := make([]string, len(runCfg.Config.PluginSelections))
```

Comment:
I just removed this section. I could have gated it by checking if the config was non-empty but it just didnt seem like a valuable piece of code.

They can immediately check the plugins via `sonobuoy status` if they want and could have checked it beforehand with sonobuoy gen. Now they can even tee the output from sonobuoy gen to a file _and_ run it with sonobuoy run so they have a true file of what they're doing.